### PR TITLE
Create database from diagram

### PR DESCRIPTION
**Pull request by fuadmoin**
In this branch we: 

- [ ] created a clinic database based on the given diagram.
- [ ] created tables that are on the diagram.
- [ ] created a join table for the many-to-many relations.

### CODE REVIEW

File: schema_based_on_diagram.sql
Comment (KDlamini) [inline schema_based_on_diagram.sql:1]: Great Job on creating all tables correctly. 💯 

Please add indexes on all foreign keys as per project requirements. 
image

For example:
```ruby
CREATE INDEX ON medical_histories (patient_id);
```

- Kindly add indexes on all other foreign keys on every table.

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi team @fuadmoin, @karanJ2212 🙋‍♀️,  

Great work so far! 💯 
You have done a great job creating a database from a diagram. 🎊 
There are some issues that you still need to work on improving your project according to the requirements. You are almost there! 💪🏾 

## Highlights 🌞
 
✔️  Professional pull request
✔️  All tables from the diagram are correctly created
✔️  The "join table" for the many-to-many relationships is created
✔️  The README is well documented

## Required Changes ♻️
 
**Kindly look through the comments below 👇 for required changes**

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+   treatment_id INTEGER ,
+    PRIMARY KEY (id),
+    CONSTRAINT "fk_invoice.id"
+    FOREIGN KEY (invoice_id)
+     REFERENCES invoices(id)
+        ON DELETE CASCADE
+);
+
+CREATE INDEX invoice_items_id_index
+ ON invoice_items (invoice_id);
```
Comment (OyePriscilla) [inline schema_based_on_diagram.sql]: - [ ] Please take note that `invoice_items` table has two foreign keys (`invoice_id and treatment_id`). Kindly create an index for `treatment_id` as well. 👍

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @fuadmoin 👏 👏, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

so-close-so-close-seth-meyers

 ## Highlights
Implemented indexes on most of the foreign keys ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by binhussen — APPROVED** [review]
Hi @cherelemma , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- Fixed Request Changes  ✔️
- Well-Structured Query ✔️
- Created based on diagram ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**